bin/key_server.js

```diff
@@ -167,6 +167,7 @@ function main() {
         op: 'promise.unhandledRejection',
         error: reason
       })
+      process.exit(8)
```

Comment:
We had this before in the auth-server, but had to remove it in prod because prod  would get `unhandledRejection`s

Reply:
This was added while narrowing down possible issues. Consider it gone!